packages/terra-menu/src/Menu.jsx

```diff
@@ -53,12 +53,17 @@ const propTypes = {
    * Otherwise, the menu will display without an arrow and right aligned.
    */
   isArrowDisplayed: PropTypes.bool,
+  /**
+   * Header Title for menu if menu contains drill-in sub-menus
```

Comment:
Could we add to this to say that if this prop is provided, but the menu doesn't contain at least one submenu, the header won't be applied.

Reply:
Updated here 11c1defc99dc75b240409155cea00a864efe808a